cadastre/classes/listParcelleLieuDatasource.class.php

```diff
@@ -27,6 +27,13 @@ public function getData($form)
         $layerId = $form->getData($this->criteriaFrom[2]);
         $section = $form->getData($this->criteriaFrom[3]);
         $voie = $form->getData($this->criteriaFrom[4]);
+
+        $result = array();
+
+        if (empty($section) && empty($voie)) {
+            return $result;
```

Comment:
```suggestion
            return array();
```

Mineur, mais je préfère cette notation, je trouve que on lit plus vite "retourne une liste vide".
Et descendre la déclaration de `$result` quand on en a vraiment besoin pour modifier la liste. On ne se pose pas la question si `$result` est modifié entre les lignes L31 et L65.

Idem pour les autres dans ce PR.

My 2 cents, feel free to skip :)

Reply:
@Gustry en gros faire comme avant ;-)